### PR TITLE
cloning enhancment and duplication

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/containers/Auth/style.scss [hunk 1]
```diff
@@ -122,6 +122,7 @@
         }
       }
     }
+
```
Comment (simonebelotti75) [inline src/containers/Auth/style.scss:125]: Unnecessary blank line

File: src/containers/Search/style.scss [hunk 2]
```diff
@@ -282,23 +282,85 @@
...
+      border-radius: 30px;  
+      width: 98px;
+      height: 40px;
+      font: normal normal normal 15px/20px Open Sans;
+      letter-spacing: 0px;
+      color: #FFFFFF;
+      border:none
+    }
   }
+
```
Comment (simonebelotti75) [inline src/containers/Search/style.scss:318]: Unnecessary blank line

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]
There are also some spacing issues
If you can also fix that, it would be great

File: src/containers/Auth/style.scss (hunk 1 above)
Comment (muhammadqamar) [reply src/containers/Auth/style.scss:125]: fixed

File: src/containers/Search/style.scss (hunk 2 above)
Comment (muhammadqamar) [reply src/containers/Search/style.scss:318]: fixed

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**